### ISSUE TITLE
Construction of datasets

### ISSUE THREAD

**Issue by PabloMosUU** [Rutgerdj/mortalityPrediction#7]
https://github.com/Rutgerdj/mortalityPrediction/blob/bf1aa48efe687eba66b9491fff726121cef47a06/train.py#L28
I think you are trying to accomplish a training (and testing) dataset with equal numbers of positives and negatives, and where the negatives have equal numbers of with-keyword and without-keyword datapoints. I question the validity of this, but I leave that for below*. If we assume that doing this is correct, then the multiplier that you need here is 2, not 3. Because you need the contribution to the training data to be 4 times as large as the contribution to the test set (80-20 split), and the test contribution here was 1/2

*As for the validity of the way you construct the dataset, there is nothing wrong per se with your dataset, but it's not a realistic dataset. Therefore, unless Buscemi did the same data selection, I don't think you can compare your results with human performance (until you make an experiment to measure human performance)

**Comment by PabloMosUU** [comment]
Also, the way you construct the datasets needs to be explained in the paper
